src/lib/eina/eina_fnmatch.h

```diff
@@ -0,0 +1,33 @@
+#ifndef EINA_FNMATCH_H
+#define EINA_FNMATCH_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#ifdef _WIN32
+
+/* Bits set in the FLAGS argument to `fnmatch'.  */
+#define FNM_PATHNAME (1 << 0) /* No wildcard can ever match `/'.  */
+#define FNM_NOESCAPE (1 << 1) /* Backslashes don't quote special chars.  */
+#define FNM_PERIOD   (1 << 2) /* Leading `.' is matched only explicitly.  */
+
+#define FNM_FILE_NAME   FNM_PATHNAME /* Preferred GNU name.  */
+#define FNM_LEADING_DIR (1 << 3)     /* Ignore `/...' after a match.  */
+#define FNM_CASEFOLD    (1 << 4)     /* Compare without regard to case.  */
```

Comment:
(Not a change request)

About those "table statements" I mentioned earlier: in this case (sequences of `#define`) it's pretty fine.